includes/polling/ports.inc.php

```diff
@@ -227,7 +227,7 @@
 if ($config['enable_ports_etherlike']) {
     echo 'dot3Stats ';
     $port_stats = snmpwalk_cache_oid($device, 'dot3StatsEntry', $port_stats, 'EtherLike-MIB');
-} else {
+} elseif ($device['os'] != 'asa') {
```

Comment:
Seems like we should really add something like:

poller_sub_modules:
    ports_dot3: 0
    ports_dot1q: 0

Into the yaml and check that as we'll have more devices to add to that if() list.

I've made those array names up btw, we need to pick something decent.